src/compiler/typescript_to_rust/generate_alias_table/debug.ts

```diff
@@ -0,0 +1,15 @@
+// TODO this file should be removed as soon as the robust imports epic is completed
+
+export const FILES_OF_INTEREST = [
```

Comment:
Create an issue and put it in the epic so that we don't forget to delete this file